docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   clamav:
     container_name: clamav
-    image: mrshow/clamav:latest
+    image: mkodockx/docker-clamav

```

Comment:
@callmekatootie just curious as to the image change. Is that one better?

Reply:
@cwdcwd I only had it changed because the dev went with an image that you had not mentioned in our discussion. I used the one that was part of the original discussion.

I don't think it offers any advantage and I tested them with both. Didn't have to make any changes. The only difference would be that the former downloaded quickly. The latter seemed to take some time though.